source/application/CMakeLists.txt

```diff
@@ -17,12 +17,21 @@ build_d_executable(
     "dextool_dextool"
 )
 
+if(UNIX AND NOT APPLE)
 add_custom_command(
     TARGET ${DEXTOOL_MAIN_EXE}
     POST_BUILD
     COMMAND cp --reflink=auto ${CMAKE_CURRENT_BINARY_DIR}/${DEXTOOL_MAIN_EXE} ${CMAKE_BINARY_DIR}/${DEXTOOL_MAIN_EXE}
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 )
+elseif(UNIX AND APPLE)
+add_custom_command(
+    TARGET ${DEXTOOL_MAIN_EXE}
+    POST_BUILD
+    COMMAND cp ${CMAKE_CURRENT_BINARY_DIR}/${DEXTOOL_MAIN_EXE} ${CMAKE_BINARY_DIR}/${DEXTOOL_MAIN_EXE}
```

Comment:
remove the branch, remove the --reflink=auto. I just tried to be "smart". lets keep it simple :)